client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./PageHeader.module.css";
+import { PropTypes } from "prop-types";
+
+const MenuDropdown = ({ onClose }) => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {
+    // TODO : API call for logout
+    navigate("/");
+  };
+
+  return (
+    <div className={styles.navDropdown}>
+      <NavButton hamburger={false} onClick={onClose} />
+      <div className={styles.menuItemContainer}>
+        <MenuDropdownItem
+          label={"Calendar"}
+          action={() => navigate("/calendar")}
+        />
+        <MenuDropdownItem
+          label={"Task History"}
+          action={() => navigate("/task_history")}
+        />
+        <MenuDropdownItem
+          label={"Insights"}
+          action={() => navigate("/insights")}
+        />
+        <MenuDropdownItem label={"Log out"} action={handleLogOut} />
+      </div>
+    </div>
+  );
+};
+
+const MenuDropdownItem = ({ label, action }) => {
+  return (
+    <button className={styles.navDropdownItem} onClick={action}>
+      {label}
+    </button>
+  );
+};
+
+const NavButton = ({ hamburger, onClick }) => {
+  return (
+    <button className={styles.navButton} onClick={onClick}>
+      {hamburger ? (
+        <FiMenu className={styles.hamburgerIcon} />
+      ) : (
+        <MdClose className={styles.closeIcon} />
+      )}
+    </button>
+  );
+};
+
+const PageHeader = ({ pageTitle }) => {
+  const [menuOpened, setMenuOpened] = useState(false);
+
+  const toggleMenu = () => {
+    setMenuOpened(!menuOpened);
+  };
+
+  return (
+    <div className={styles.calendarHeaderContainer}>
```

Comment:
Minor comment: There can probably be more padding/margin on the Header. If not, the padding can also be on the task_history page. The other pages seem fine. It's just the task_history page that seems a bit too close to the top of the page

![image](https://user-images.githubusercontent.com/24628243/165217925-9f1ea2e3-b313-4025-9163-90f9e227d5d4.png)

Reply:
Added an additional 10px spacing between the header and filters